cli/api/storageoptions.go

```diff
@@ -36,7 +36,7 @@ func getDefaultStorageOptions(driverType volume.DriverType, config utils.ConfigR
 		addStorageOption(config, "DM_THINPOOLDEV", "", func(v string) {
 			options = append(options, fmt.Sprintf("dm.thinpooldev=%s", v))
 		})
-		addStorageOption(config, "DM_BASESIZE", "100G", func(v string) {
+		addStorageOption(config, "DM_BASESIZE", "", func(v string) {
```

Comment:
I thought https://github.com/control-center/serviced/blob/develop/volume/devicemapper/devicemapper.go#L317 would set it to 100G anyway, but rechecked by base size and found `10737418240` which looks similar to `107374182400` if you're not paying attention.